worker-document-testing-unit/src/test/java/com/hpe/caf/worker/document/testing/TestDocumentWorker.java

```diff
@@ -19,7 +19,13 @@
 import com.hpe.caf.api.worker.DataStoreException;
 import com.hpe.caf.worker.document.exceptions.DocumentWorkerTransientException;
 import com.hpe.caf.worker.document.extensibility.DocumentWorker;
-import com.hpe.caf.worker.document.model.*;
```

Comment:
All of the guidelines (Google, Sun, Checkstyle) say to avoid star imports:
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport

Reply:
My change removed the star imports. This is because I modified my intellij settings.